package.json

```diff
@@ -1,7 +1,7 @@
 {
   "title": "smoothscroll",
   "name": "smoothscroll-polyfill",
-  "version": "0.3.5",
+  "version": "0.3.6",
```

Comment:
Avoid pushing versions. Just add changes to the src file, we will handle the releases on our end.